public/app/features/panel/components/PanelRenderer.tsx

```diff
@@ -151,7 +152,9 @@ export function useFieldOverrides(
         data: series,
         fieldConfig,
         fieldConfigRegistry,
-        replaceVariables: (str: string) => str,
+        replaceVariables: (str: string, scopedVars?: ScopedVars) => {
```

Comment:
is `str` here the JSON string of the full dataframe? and what does the `.replace()` function body typically look like?

is it possible to attach a simple dashboard to this PR that triggers this code path?

Reply:
This gets passed to `applyFieldOverrides` then to `getLinksSupplier` which will call it on parts of the link config (depending on if it's internal, defines an onClick handler, or not), so not the full dataframe, just individual strings like the link title or href. Specifically for what I'm needing, it gets run in `mapInternalLinkToExplore` defined in `grafana-data/src/utils/dataLinks.ts` which calls it on the title and query object. 

`replace()` is defined in the template server which in this case is defined in `public/app/features/templating/template_srv.ts`

Here's the related code from the app o11y plugin. The relevant link is in `topExceptions.ts L30`: https://github.com/grafana/app-o11y-kwl/pull/76/files